app/main.py

```diff
@@ -1,3 +1,28 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer
+from app.car import Car
+
+
+def shop_trip() -> None:
+    with open(r"app/config.json") as file:
```

Comment:
The path to the JSON file should use double quotes instead of the raw string literal. According to the checklist, use one style of quotes in your code, and double quotes are preferable.